templates-global/left-sidebar-check.php

```diff
@@ -8,11 +8,18 @@
 // Exit if accessed directly.
 defined( 'ABSPATH' ) || exit;
 
-$sidebar_pos = get_theme_mod( 'uds_wp_sidebar_position' );
-
-if ( 'left' === $sidebar_pos || 'both' === $sidebar_pos ) {
-	get_template_part( 'templates-sidebar/sidebar', 'left' );
-}
 ?>
-
-<div class="col-md content-area" id="primary">
+<!-- ========= Left Sidebar ============ -->
+		<?php
+		global $uds_wp_sidebar_opts;
+			  global $content_classes;
+		if ( 'left' == $uds_wp_sidebar_opts['use_sidebar'] ) :
+			   $content_classes = 'col-md col-xs-12 pr-0';
+			get_template_part( 'templates-sidebar/sidebar', 'left' );
+			?>
```

Comment:
Unnecessary php tags here that just inserts extraneous carriage returns into the HTML output